quill-sql/src/main/scala/io/getquill/context/sql/norm/ExpandMappedInfix.scala

```diff
@@ -0,0 +1,12 @@
+package io.getquill.context.sql.norm
+
+import io.getquill.ast._
+
+object ExpandMappedInfix {
```

Comment:
I am afraid this will break  user defined `infix.map(a => f(a))` call

Reply:
@jilen Thanks for a review? Could you post an example? There's a test which handles that https://github.com/getquill/quill/pull/1131/commits/6a9460317a98989e56a6d0215a852c7de2e4e8ff#diff-e564af4387151d9877a17343b828b795R1050. `ExpandMappedInfix` is applied before the latest `Normalize` so user defined map will be normalized with expanded map in `ExpandMappedInfix`. However if there's any known case please share and I will include them in test

Reply:
@mentegy  What about postgresql's `Factorial` (or any other postfix operator)

```scala
infix("$i !").map(x => f(x))       
```
Will this transformation break such operator ?